homeassistant/components/rfxtrx/sensor.py

```diff
@@ -64,15 +65,15 @@ def supported(event):
         return isinstance(event, (ControlEvent, SensorEvent))
 
     entities = []
-    for packet_id in discovery_info[CONF_DEVICES]:
+    for packet_id, entity in discovery_info[CONF_DEVICES].items():
```

Comment:
Let's rename `entity` to `entity_info` like in the other platforms. The `entity` variable name is reserved for `Entity` instances.

Reply:
I'll clean it up. I actually copied it from binary sensor, I will check if there are more of these.